### DISCUSSION TITLE
ettercap issue

### DISCUSSION THREAD

**Discussion by bellakamsar** [community/community#132269] [Programming Help]
### Select Topic Area

Question

### Body

                                                                                                                                                            
┌──(camus㉿kali2)-[~/Desktop]
└─$ sudo ettercap -G -w packets.pcap

ettercap 0.8.3.1 copyright 2001-2020 Ettercap Development Team

Ooops ! This shouldn't happen...
Segmentation Fault...

Please recompile in debug mode, reproduce the bug and send a bugreport

 Have a nice day!

**Comment by LiteBrite82** [comment]
Hi @bellakamsar ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as the [ettercap repository](https://github.com/Ettercap/ettercap). 

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
